com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/VertMesh.hlsl

```diff
@@ -140,9 +140,9 @@ VaryingsMeshType VertMesh(AttributesMesh input)
     output.normalWS = normalWS;
     output.tangentWS = tangentWS;
 #endif
-#if defined(DEBUG_DISPLAY) && (SHADERPASS == SHADERPASS_FORWARD || SHADERPASS == SHADERPASS_FORWARD_UNLIT)
-        if (_DebugFullScreenMode == FULLSCREENDEBUGMODE_VERTEX_DENSITY)
-            IncrementVertexDensityCounter(output.positionCS);
+#if defined(SHADERPASS) && (SHADERPASS == SHADERPASS_FULLSCREEN_DEBUG)
```

Comment:
you don't need to check for #if defined(SHADERPASS) 
you can just check
#if SHADERPASS == SHADERPASS_FULLSCREEN_DEBUG

Reply:
i didn't look too much into it but it looks like the file is included somewhere SHADERPASS is not defined